examples/specs/wheat_wages.vl.json

```diff
@@ -16,7 +16,7 @@
   },
   "layer": [
     {
-      "mark": {"type": "bar", "fill": "#aaa", "stroke": "#999"},
+      "mark": {"type": "bar", "fill": "#aaa", "stroke": "#999", "orient": "vertical"},
```

Comment:
@kanitw For this one, I will keep as "bar" but change to add "binned" on encoding.x instead of the "orient" since this spec has a kind of the bin transform implying it is binned.